src/python/benchUtil.py

```diff
@@ -1381,17 +1399,26 @@ def simpleReport(self, baseLogFiles, cmpLogFiles, jira=False, html=False, baseDe
       else:
         w('%16s' % ('%7.1f%% (%4d%% - %4d%%)' % (psAvg, psWorst, psBest)))
 
+      if jira:
+        w('|%s' % (jiraColor(pValue)))
```

Comment:
Maybe we could use colors (does jira allow colors?  html certainly does) to indicate tiers of confidence?  Or we can do on followon PR!